tests/rptest/services/redpanda.py

```diff
@@ -30,23 +30,12 @@
 
 
 class RedpandaService(Service):
-    PERSISTENT_ROOT = "/mnt/redpanda"
+    PERSISTENT_ROOT = "/var/lib/redpanda"
     DATA_DIR = os.path.join(PERSISTENT_ROOT, "data")
-    CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "redpanda.yaml")
+    CONFIG_FILE = "/etc/redpanda/redpanda.yaml"
     STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "redpanda.log")
     CLUSTER_NAME = "my_cluster"
     READY_TIMEOUT_SEC = 10
-    V_DEV_MOUNT = "/opt/v"
-
-    # ducktape `--globals` for selecting build
-    BUILD_DIR_KEY = "redpanda_build_dir"
-    DEFAULT_BUILD_DIR = "vbuild"
-    BUILD_TYPE_KEY = "redpanda_build_type"
-    DEFAULT_BUILD_TYPE = "release"
-    COMPILER_KEY = "redpanda_compiler"
-    DEFAULT_COMPILER = "clang"
-    PACKAGING_KEY = "redpanda_packaging"
-    DEFAULT_PACKAGING = "dir"
```

Comment:
are we still retaining the ability to test the different build types?

Reply:
yeah, that's handled by `vtools` now. There's an `rp_install_path` variable that the `vtools test ducky` command defines that ends up directing which folder to reference in tests (the new `find_binary()` logic reflects this. In general, the assumption is that theres a `f"{rp_install_path}/redpanda/" folder for the RP installation (and a `pandaproxy` one)